x-pack/plugins/security_solution/public/common/components/alert_count_by_status/translations.ts

```diff
@@ -20,7 +20,7 @@ export const ALERTS_BY_RULE = i18n.translate(
 export const COLUMN_HEADER_RULE_NAME = i18n.translate(
   'xpack.securitySolution.alertCountByRuleByStatus.ruleName',
   {
-    defaultMessage: 'kibana.alert.rule.name',
+    defaultMessage: 'Rule name',
```

Comment:
While you're in here, would you mind also capitalizing `COLUMN_HEADER_COUNT` on L30. Thanks!

Reply:
Thank you!